.env

```diff
@@ -5,7 +5,7 @@ MODEL_NAME_2="your_second_model.pt"
 MODEL_ALLOWED_CLASSES="0"
 MODEL_2_ALLOWED_CLASSES="0"
```

Comment:
we no longer need to have the model settings, we use project instead (which under the hood implements the models)

Reply:
Refactored